src/redux/books/book.js

```diff
@@ -1,19 +1,85 @@
-// Actions
+// URL API
+const URL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi';
+const URLid = '/apps/tdUGvbgxfHyC017dl4Kz/books';
+const BASE_URL = `${URL}${URLid}`;
+
+// Action Fetch book - API FETCH
+const FETCH_BOOK = 'FETCH_BOOK';
+// fetchBook Action creator
```

Comment:
- [ ] Congrat for pushing to this level. But you need to follow the requirements of your project because it's for a purpose
- **Here is the requirement**: Use [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) or [axios](https://axios-http.com/docs/intro) with [createAsyncThunk](https://redux-toolkit.js.org/api/createAsyncThunk) for making API calls.